draft-wang-ppm-differential-privacy.md

```diff
@@ -884,65 +910,129 @@ in order to achieve various combinations of `(EPSILON, DELTA)`-DP.
 
 ### Prio3Histogram with Aggregator-DP
 
-> TODO Describe a policy that is compatible with Prio3Histogram and that
-> targets the OAOC trust model.
+Aggregator-DP requires Aggregators to add noise to their aggregate shares before
+outputting them. Under OAOC trust model, we can achieve good `EPSILON`-DP, or
+`(EPSILON, DELTA)`-DP, as long as at least one of the Aggregators is honest. The
+amount of noise needed by Aggregator-DP typically depends on the target DP
+parameters `EPSILON`, `DELTA`, and also the `SENSITIVITY` of the aggregation
+function.
+
+In this policy, we will describe how to achieve `(EPSILON, DELTA)`-DP, by asking
+each Aggregator to independently add discrete Gaussian noise to its aggregate
+share, and by using `Prio3Histogram` VDAF for aggregation and robustness
+considerations.
+
+#### Discrete Gaussian Mechanism for Aggregator-DP
+
+The Aggregator-DP we will use here is the discrete Gaussian mechanism described
+in {{discrete-gaussian}}, which has mean of 0, and is initialized with a `SIGMA`
+parameter that stands for the standard deviation of the Gaussian distribution.
+In order to achieve `(EPSILON, DELTA)`-DP under OAOC trust model, all
+Aggregators need to independently add discrete Gaussian noise to all coordinates
+of their aggregate shares.
+
+To compute the parameter `SIGMA` for discrete Gaussian, Theorem 7 in {{CKS20}}
+and Theorem 8 in {{BW18}} have shown that the `SIGMA` parameter can be tuned to
```

Comment:
Why are two citations needed/useful?

Reply:
So Theorem 8 of [BW18](https://arxiv.org/pdf/1805.06530.pdf) is the first paper that has the analysis for why *continuous* Gaussian achieves `(EPSILON, DELTA)`-DP. Theorem 7 in [CKS20](https://arxiv.org/pdf/2004.00010.pdf) shows similar result for discrete Gaussian.

There is a caveat though, also related to one feedback from Kunal: CKS20 makes a note that: when `SIGMA` is small, there is small difference in `(EPSILON, DELTA)` achieved by continuous and discrete Gaussian (Figure 1 has some details). If we use the algorithm from BW18, we need to be careful when `SIGMA` is small, i.e., when the desired `(EPSILON, DELTA)`-DP is weak. I made an author note in the draft.

Reply:
Good enough for now.